src/main/java/Deadline.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * A Task with a deadline.
+ * @author Thomas Hogben
+ */
+public class Deadline extends Task {
+    private String deadline;
+
+    /**
+     * @param description The description of the Task.
```

Comment:
Missing out the header paragraphs in your javadocs 🤔 , perhaps a short summary of what the method does will be nice, even if it is quite straightforward.